justfile

```diff
@@ -0,0 +1,57 @@
+[doc('Build all targets in the project')]
+build-all:
+  bazel build //...
+
+[doc('Build all JS/TS files')]
+build-js:
+    bazel build -- $(bazel query "kind(npm_package, //...)" --output label 2>/dev/null | tr '\n' ' ')
+
+[doc('Build all core JS/TS files')]
+build-core:
+    bazel build //core/...
+
+[doc('Test targets in the project')]
+test-all:
+  bazel test //...
+
+[doc('Test all JS/TS files')]
+test-js:
+  bazel test -- $(bazel query "kind(js_test, //...)" --output label 2>/dev/null | tr '\n' ' ')
+
+[doc('Test all core JS/TS files')]
+test-core:
+    bazel test //core/...
+
+[doc('Lint all JS/TS files')]
+lint-js:
```

Comment:
Do we need to have language discrimination for common language things? Practically for performance, maybe, but if I want to lint the project, ideally I'd just lint the whole project.

Reply:
I don't see any downside to having per language targets in addition to global targets.

Reply:
I feel like people are more likely to want to lint smaller subsets of the project based on what they're updating rather than the full project all the time